WordPress/UITestsFoundation/Screens/CommentsScreen.swift

```diff
@@ -0,0 +1,63 @@
+import ScreenObject
+import XCTest
+
+public class CommentsScreen: ScreenObject {
+
+    private let replyTextViewGetter: (XCUIApplication) -> XCUIElement = {
+        $0.textViews["ReplyText"]
+    }
+
+    private let backButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.navigationBars.buttons["Reader"]
+    }
+
+    private let replyButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Reply"]
+    }
+
+    var replyTextView: XCUIElement { replyTextViewGetter(app) }
+    var backButton: XCUIElement { backButtonGetter(app) }
+    var replyButton: XCUIElement { replyButtonGetter(app) }
+
+    init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            expectedElementGetters: [
+                // swiftlint:skip:next opening_brace
+                { $0.navigationBars["Comments"] },
+                replyTextViewGetter
+            ],
+            app: app,
+            waitTimeout: 7
+        )
+    }
+
+    public static func isLoaded() -> Bool {
+        (try? ReaderScreen().isLoaded) ?? false
+    }
+
+    public func navigateBack() throws -> ReaderScreen {
+        backButton.tap()
+        return try ReaderScreen()
+    }
+
+    @discardableResult
+    public func replyPost(_ comment: String) -> CommentsScreen {
```

Comment:
What do you think of 

```suggestion
    public func replyToPost(_ comment: String) -> CommentsScreen {
```

or 

```suggestion
    public func commentOnPost(_ comment: String) -> CommentsScreen {
```

Reply:
Yep, done it on [445c3e9](https://github.com/wordpress-mobile/WordPress-iOS/pull/20432/commits/445c3e9e712d2692489ed3a3c8c252b56a9287ec). 🙇‍♂️